tests/integration/data/test_data_a11y.json

```diff
@@ -0,0 +1,63 @@
+{
+    "zones": [
+        {
+            "index": 1,
+            "title": "Zone 1",
+            "description": "This describes zone 1",
```

Comment:
I'm worried about having too many fixture files, which arguably makes the tests harder to maintain when we change the data format. Could you just add a `description` to one of the existing fixtures? (unless that would break existing tests)